game_template/lib/main.dart

```diff
@@ -2,8 +2,12 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
-import 'package:firebase_crashlytics/firebase_crashlytics.dart';
-import 'package:flutter/foundation.dart';
+import 'dart:io' show Platform;
+
+import 'package:firebase_core/firebase_core.dart' show Firebase, FirebaseOptions;
```

Comment:
Why do you need the `show X` on the imports? I was able to successfully uncomment and add these imports:

```
import 'dart:io';

import 'package:firebase_core/firebase_core.dart';
import 'package:firebase_crashlytics/firebase_crashlytics.dart';
import 'package:flutter/foundation.dart';

import 'firebase_options.dart';
```

Reply:
John - I wasn't sure why they were necessary but it was the only way I could get rid of the errors at the time.
It's working for me now without the `show X` - not sure what's changed.

Brett - thanks for the suggestion. I went with the "add them commented out, with instructions to uncomment them if Crashlytics is added."